scripts/missions/sandoria/6_2_Ranperres_Final_Rest.lua

```diff
@@ -16,6 +16,12 @@ local krtID = zones[xi.zone.KING_RANPERRES_TOMB]
 
 local mission = Mission:new(xi.mission.log_id.SANDORIA, xi.mission.id.sandoria.RANPERRES_FINAL_REST)
 
+local ranparesRestTable =
+{
+    krtID.mob.CORRUPTED_YORGOS,
+    krtID.mob.CORRUPTED_SOFFEIL,
+    krtID.mob.CORRUPTED_ULBRIG,
+}
 mission.reward =
```

Comment:
Nitpick: can you add a space between these :)

Reply:
Anything for you my friend.